peoplefinder/templates/peoplefinder/team-people.html

```diff
@@ -12,6 +12,10 @@ <h1 class="govuk-heading-l">{{ heading }}</h2>
 
 {% include 'peoplefinder/components/pagination.html' with page=team_members page_numbers=page_numbers page_obj_name='people' %}
 
+{% if team_members %}
+    <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="#" hx-get="{% url 'team-email-members' team.slug %}" hx-target="closest p">Get email all members link</a></p>
```

Comment:
```suggestion
    <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="#" hx-get="{% url 'team-email-members' team.slug %}" hx-target="closest p">Get link to email all members</a></p>
```
slightly less clunky copy?

Reply:
agreed